entgql/annotation.go

```diff
@@ -59,9 +59,10 @@ type (
 
 	// DirectiveArgument return a GraphQL directive argument
 	DirectiveArgument struct {
-		Name  string        `json:"name,omitempty"`
-		Value string        `json:"value,omitempty"`
-		Kind  ast.ValueKind `json:"kind,omitempty"`
+		Name     string             `json:"name,omitempty"`
+		Children ast.ChildValueList `json:"children,omitempty"`
+		Value    string             `json:"value,omitempty"`
+		Kind     ast.ValueKind      `json:"kind,omitempty"`
```

Comment:
Maybe we can squash these 3  into an `ast.Value` and compose this object on the option.

Reply:
I honestly think it makes sense, will make it more flexible. In terms of changes, do you mean that instead of duplicating each field from `ast.Value` on `DirectiveArgument` use an actual `ast.Value` as a field or remove the `DirectiveArgument` and use `ast.Value` altogether?

Reply:
Yes, thanks.

Reply:
@a8m `DirectiveArgument` was renamed to `DirectiveListArgument` as my suggestion. `DirectiveListArgument` is a helper that allows ppl build arguments quickly and with less code. If they want to customize more, just build the Children manually and pass it to the directive argument.

Reply:
Here's an usage example for the @giautm suggestion: https://github.com/ent/contrib/pull/380#discussion_r977438993

Reply:
> Yes, thanks.

I got your idea here. But, it's a breaking change. Do we need to break? if yes, we can use `ast.Argument` to replace `entgql.DirectiveArgument `